.cookiecutter/includes/tox/setenv

```diff
@@ -4,6 +4,7 @@ dev: BROKER_URL = {env:BROKER_URL:amqp://guest:guest@localhost:5674//}
 dev: FEATURE_FLAGS_COOKIE_SECRET = {env:FEATURE_FLAGS_COOKIE_SECRET:notasecret}
 dev: FEATURE_FLAGS_ALLOWED_IN_COOKIE = {env:FEATURE_FLAGS_ALLOWED_IN_COOKIE:vitalsource}
 dev: H_AUTHORITY = {env:H_AUTHORITY:lms.hypothes.is}
+dev: REGION_CODE = {env:REGION_CODE:us}
```

Comment:
We'll have to add this new `REGION_CODE` envvar to all environments before merging this PR.